kyuubi-common/src/test/scala/org/apache/kyuubi/KyuubiFunSuite.scala

```diff
@@ -134,6 +134,23 @@ trait KyuubiFunSuite extends AnyFunSuite
     }
   }
 
+  final def withSystemProperty(properties: Map[String, String])(f: => Unit): Unit = {
+    val originValues = properties.map {
+      case (key, value) =>
+        val originValue = System.getProperty(value)
```

Comment:
`System.getProperty(value)` value? do you mean key?

Reply:
> `System.getProperty(value)` value? do you mean key?

yeah, i wiil fix it